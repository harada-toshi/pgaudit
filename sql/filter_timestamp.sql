\! cp sql/filter_timestamp/filter_timestamp.conf /tmp/config.conf
\! echo "pgaudit.config_file = '/tmp/config.conf'" >> ./tmp_check/data/postgresql.conf
\! rm ./tmp_check/tmp_check.log
\! pg_ctl restart -w -D ./tmp_check/data -l ./tmp_check/tmp_check.log
\! psql contrib_regression -q -a < sql/filter_timestamp/filter_timestamp.sql
\! rm /tmp/config.conf
\! grep "AUDIT:" ./tmp_check/tmp_check.log | sed 's/[0-9]\+/X/g' > results/filter_timestamp.out

