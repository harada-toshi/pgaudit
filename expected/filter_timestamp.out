	        format = 'AUDIT:(A) %t,%class,%command_tag,%command_text'
	        format = 'AUDIT:(A) %t,%class,%command_tag,%command_text'
	        format = 'AUDIT:(B) %t,%class,%command_tag,%command_text'
	        format = 'AUDIT:(B) %t,%class,%command_tag,%command_text'
LOG:  AUDIT:(A) X-X-X X:X:X     JST,READ,SELECT,SELECT now()
LOG:  AUDIT:(B) X-X-X X:X:X     JST,READ,SELECT,SELECT now()
